### PR TITLE
Added macOS and windows CI

### PR DESCRIPTION
**Pull request by michaelperel**
This PR adds an Azure Pipeline that ensures installation on macOS and windows (the service is offered for free). By default, it will run on all PRs to master. The pipeline is defined in `azure-pipelines.yml`, and currently connected to my fork.

If this PR is accepted, a new pipeline needs to be setup for ryan-roemer/django-cloud-browser (I do not have the credentials, so could only add the pipeline to my fork). Essentially all that needs to be done is clicking a few buttons in the UI, and the status badge needs to be updated as well. Instructions for adding the pipeline can be found in the first video on [this page](https://docs.microsoft.com/en-us/azure/devops/pipelines/get-started/?view=azure-devops). The updated links for the status badge can be found by following [these instructions](https://docs.microsoft.com/en-us/azure/devops/pipelines/create-first-pipeline?view=azure-devops&tabs=tfs-2018-2#add-a-status-badge-to-your-repository).

If you have any questions or need any help, please let me know. Thanks @c-w for helping with the pipeline :)

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -66,5 +66,8 @@ a long list of results.
 .. |build| image:: https://travis-ci.org/ryan-roemer/django-cloud-browser.svg
    :target: https://travis-ci.org/ryan-roemer/django-cloud-browser
 
+.. |install| image:: https://dev.azure.com/michaelsethperel/django-cloud-browser/_apis/build/status/michaelperel.django-cloud-browser?branchName=master
```
Comment (c-w) [inline README.rst]: Consider renaming `|build|` to `|travis|` and `|install|` to `|azdo|` (or similar) to make naming more explicit.

File: .travis.yml [hunk 2]
```diff
@@ -8,7 +8,10 @@ install:
   - pip install -e .[test]
 
 before_install:
-  - sudo rm -f /etc/boto.cfg
+  - sudo rm -f /etc/boto.cfg # Workaround for travis boto issue with GCE
+  - python -m pip install --upgrade pip
+  # easy_install instead of pip because of a problem with travis' virtualenv
+  # https://stackoverflow.com/questions/35780537/error-no-module-named-markerlib-when-installing-some-packages-on-virtualenv
```
Comment (c-w) [inline .travis.yml]: Consider shortening the URL to https://stackoverflow.com/q/35780537/3817588

**Comment by c-w** [comment]
@ryan-roemer Looks like I don't have permissions to add apps to the repo:

Screenshot showing AzDO pipeline import error

**Comment by c-w** [comment]
Merging this for now; let's make a second PR to update the build bade links once the pipeline is set up.

**Comment by ryan-roemer** [comment]
@c-w -- I'll come back to this when I get a moment to create a one-off GitHub user to limit the oauth scope of the grant for Azure Pipelines GH authorization as I'm a member of a ton of orgs including a lot of our client ones.

I've added https://github.com/ryan-roemer/django-cloud-browser/issues/26 to track

**Pull request merged**